cluster/images/hyperkube/kube-proxy.json

```diff
@@ -13,7 +13,8 @@
                 "proxy",
                 "--master=http://127.0.0.1:8080",
                 "--v=2",
-                "--resource-container=\"\""
+                "--resource-container=\"\"",
+                "--proxy-mode=iptables"
```

Comment:
Why enable iptables proxy mode? It's not enabled by default in regular clusters?